src/runners/runner.js

```diff
@@ -0,0 +1,60 @@
+
+const uuid4 = require('uuid4');
+const {
+    utils,
+    eventInterface,
+    event: serverlessEvent,
```

Comment:
I think `event` is a reserved word, don't use it

Reply:
I've checked it, And it's not a reserved word.